examples/AspNetCore/README.md

```diff
@@ -0,0 +1,68 @@
+# OpenTelemetry Example Application
+
+This project is an example of ASP.NET Core instrumentation for tracing and
+logging.
+
+The following exporters are configured for viewing the traces:
+
+* Console
+* Zipkin
+* Otlp
+* Jaeger
+
+The following exporters are configured for viewing the metrics:
+
+* Console
+* Otlp
+* Prometheus
+
+## Running the example
+
+Running instances of the following services are required to view exports:
```

Comment:
nit:

```suggestion
Running instances of the following services are required to view the exported data:
```

Reply:
@reyang Updated!

For my own reference, what does `nit` mean?

Reply:
_nit-picking_ - basically "it's a very tiny/trivial suggestion, you decide if you want to take or ignore it" 😄

Reply:
Gotcha, thanks!